pkgs/sdk/server/contract-tests/README.md

```diff
@@ -6,4 +6,4 @@ To run these tests locally, run `make contract-tests` from the SDK project root
 
 Or, to test against an in-progress local version of the test harness, run `make start-contract-test-service` from the SDK project root directory; then, in the root directory of the `sdk-test-harness` project, build the test harness and run it from the command line.
 
-Currently, the project does _not_ automatically detect the available target frameworks. It will default to building and running for .NET Core 3.1. To use a different target framework, set the environment variable `TESTFRAMEWORK` to the name of the application runtime framework (such as `netcoreapp3.1`), and set the environment variable `BUILDFRAMEWORKS` (note the S at the end) to the target framework that the SDK should be built for (which may or may not be the same).
+Currently, the project does _not_ automatically detect the available target frameworks. It will default to building and running for .NET 8.0. To use a different target framework, set the environment variable `TESTFRAMEWORK` to the name of the application runtime framework (such as `net8.0`), and set the environment variable `BUILDFRAMEWORKS` (note the S at the end) to the target framework that the SDK should be built for (which may or may not be the same).
```

Comment:
Not for this PR, but I wonder if we could change these markdown files to be interpolated or automatically updated.